scripts/deployment-helpers.sh

```
@@ -0,0 +1,161 @@
+wait_for_csv_with_min_version() {
+  local operator_prefix="$1"
+  local min_version="$2"
+  local namespace="${3:-kuadrant-system}"
+  local timeout="${4:-180}"
+  
+  echo "⏳ Looking for ${operator_prefix} (minimum version: ${min_version})..."
+  
+  local end_time=$((SECONDS + timeout))
+  
+  while [ $SECONDS -lt $end_time ]; do
+    local csv_name=$(find_csv_with_min_version "$operator_prefix" "$min_version" "$namespace")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)